src/configure/configure.ts

```diff
@@ -121,34 +122,45 @@ class Orchestrator {
             await this.getAzureSession();
             await this.getSelectedPipeline();
 
-            if (this.inputs.pipelineConfiguration.template.label === "Containerized application to AKS") {
-                // try to see if node corresponds to any parameter of selected pipeline.
-                if (resourceNode) {
-                    let resourceParam = TemplateParameterHelper.getMatchingAzureResourceTemplateParameter(resourceNode, this.inputs.pipelineConfiguration.template.parameters);
-                    if (resourceParam) {
-                        this.inputs.pipelineConfiguration.params[resourceParam.name] = resourceNode;
+            
+            if(this.inputs.pipelineConfiguration.templateInfo) {
```

Comment:
We should see if we can merge the AKS and this new flow for showing parameter inputs. If I am not wrong input definition for both local and remote templates are pretty much same except for dataSource. So,it should be possible to use inputUxDescriptor for both kind of templates - dataSource can again be abstracted as suggested above.

Reply:
We will do this when working on AKS, this PR handles the getting inputs to the template parameters passed to InputControlProvider